homework1/task05.py

```diff
@@ -20,7 +20,7 @@ def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
         maximum sum of a subarray with at-least k elements.
     """
     if len(nums) == 0 or k <= 0 or k > len(nums):
-        raise ValueError
+        raise ValueError("k should not be greater then the size of list of integers")
```

Comment:
When you have multiple conditions, it's more concise to say what values `k` can take and print the actual value of `k`. Otherwise, you need to mention all of the conditions.